localPackages/site_developer/Resources/Private/Layouts/Page/Default.html

```diff
@@ -0,0 +1,27 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
+    data-namespace-typo3-fluid="true">
+
+    <div class="header d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 border-bottom shadow-sm">
+        <h5 class="my-0 mr-md-auto font-weight-normal"><a href="{menu.0.link}" class="text-dark">{menu.0.title}</a></h5>
```

Comment:
Please also let's use indentation here, i.e., the A tag should go on a separate line, and its contents on another line. Ditto for the H1, P etc. below.